Phase5/SearchEngine/TestProject1/InvertedIndexTest.cs

```diff
@@ -0,0 +1,172 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using NSubstitute;
+using SearchEngine;
+using SearchEngine.Interfaces;
+using Xunit;
+
+namespace TestProject1
+{
+    public class InvertedIndexTest
+    {
+        private const string Path = "../../../TestDocs/DocsForTest";
+
+        
+        private Dictionary<string, string> ReturningDictionary()
+        {
+            var toReturn = new Dictionary<string, string>
+            {
+                {"firstFile", "Hello Everyone This Is Just For Test Hello! a cat is here"}
+            };
+            return toReturn;
+        }
+        
+        
+        [Fact]
+        public void InvertedIndexSimpleTest()
+        {
+            IFileReader fileReader = Substitute.For<IFileReader>();
+            fileReader.ReadingFiles(Path).Returns(ReturningDictionary());
+            InvertedIndex invertedIndex = (InvertedIndex)new InvertedIndex().TokenizeFiles(fileReader.ReadingFiles(Path));
+
+            Assert.Contains("firstFile", invertedIndex.TokenizedWords["hello"]);
+            Assert.False(invertedIndex.TokenizedWords.ContainsKey("ali"));
+        }
+        
+        
+        [Fact]
+        public void QueryTest (){
+            IFileReader fileReader = Substitute.For<IFileReader>();
+            fileReader.ReadingFiles(Path).Returns(ReturningDictionary());
+            InvertedIndex invertedIndex = (InvertedIndex)new InvertedIndex().TokenizeFiles(fileReader.ReadingFiles(Path));
+
+            UserInput userInput = new UserInput("everyone");
+            SortedSet<string> files = invertedIndex.Query(userInput);
+            Assert.False(!files.Any());
+            Assert.Equal("firstFile", files.ToList()[0]);
+            Assert.Single(files);
+        }
+        
+        
+        [Fact]
+        public void QueryTest2 (){
+            IFileReader fileReader = Substitute.For<IFileReader>();
+            fileReader.ReadingFiles(Path).Returns(ReturningDictionary());
+            InvertedIndex invertedIndex = (InvertedIndex)new InvertedIndex().TokenizeFiles(fileReader.ReadingFiles(Path));
```

Comment:
these test method names do not show the case which is under test. It is better to rename them to a more meaningful name.